Source/NSSplitViewController.m

```diff
@@ -0,0 +1,196 @@
+/* Implementation of class NSSplitViewController
+   Copyright (C) 2020 Free Software Foundation, Inc.
+   
+   By: Gregory John Casamento
+   Date: Mon 20 Jul 2020 12:55:02 AM EDT
+
+   This file is part of the GNUstep Library.
+   
+   This library is free software; you can redistribute it and/or
+   modify it under the terms of the GNU Lesser General Public
+   License as published by the Free Software Foundation; either
+   version 2.1 of the License, or (at your option) any later version.
+   
+   This library is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+   Lesser General Public License for more details.
+   
+   You should have received a copy of the GNU Lesser General Public
+   License along with this library; if not, write to the Free
+   Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+   Boston, MA 02110 USA.
+*/
+
+#import <Foundation/NSArray.h>
+#import <Foundation/NSArchiver.h>
+
+#import "AppKit/NSSplitView.h"
+#import "AppKit/NSSplitViewController.h"
+#import "AppKit/NSSplitViewItem.h"
+#import "AppKit/NSView.h"
+
+#import "GSFastEnumeration.h"
+
+@interface NSView (__NSSplitViewController_Notifications__)
+- (void) _viewWillMoveToWindow: (NSWindow *)w;
+- (void) _viewWillMoveToSuperview: (NSView *)v;
+- (void) _viewDidMoveToWindow;
+@end
+
+@interface NSView (__NSSplitViewController_Private__)
+- (void) insertSubview: (NSView *)sv atIndex: (NSUInteger)idx;
+@end
+
+@implementation NSView (__NSSplitViewController_Private__)
```

Comment:
I really would prefer to have the implementation of this method over in NSView.m. We could move the declaration into one of the AppKit private header files and include that in both places.

Reply:
Okay, doing that.

Reply:
Moved